presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -115,6 +116,7 @@
     public static final String FILE_RENAMING_ENABLED = "file_renaming_enabled";
     public static final String PREFER_MANIFESTS_TO_LIST_FILES = "prefer_manifests_to_list_files";
     public static final String MANIFEST_VERIFICATION_ENABLED = "manifest_verification_enabled";
+    public static final String METASTORE_USER_SUPPLIED_EXTRA_PARAM = "metastore_user_supplied_extra_param";
```

Comment:
nit: Usually we don't abbreviate (`PARAM` -> `PARAMETER`)

Reply:
It's not a map it's a string value. I don't want to dictate the structure of the string.